pipeline_steps/irr_role_tests.groovy

```diff
@@ -20,9 +20,14 @@ def run_irr_tests() {
                   ]
                 ]
               ]) // checkout
-              sh """#!/bin/bash
-              bash ./run_tests.sh
-              """
+              withCredentials(common.get_cloud_creds()){
```

Comment:
if you only need cloud creds to get the maas token and url, then this isn't needed as maas.maas_utils already binds common.get_cloud_creds.